kubernetes/local/gateway.sh

```diff
@@ -3,21 +3,11 @@
 
 if [[ ${1} == "up" ]]; then
 
-    if [ -f `which uname` ]; then
-        if [ `uname -p` == "arm" ]; then
-            # Apple Silicon
-            APPLE_SILICON=1
-        fi
-    fi
-
-    if [ ${APPLE_SILICON:-0} -eq 0 ]; then
-			docker pull docker.uncharted.software/auth/httpd-openidc:dev-1.0.1
-			docker pull docker.uncharted.software/auth/keycloak:dev-1.1.0
-    else
-        echo "not pulling latest images as Apple Silicon httpd-openidc has not been pushed to the repo"
-    fi
-
+		# Manually pulling images here, as the K8s sometimes timeout
+		docker pull ghcr.io/unchartedsoftware/httpd-openidc:0.1.2
+		docker pull ghcr.io/unchartedsoftware/keycloak:0.1.2
 		docker pull docker.uncharted.software/auth/terarium-theme:0.0.1
```

Comment:
Why this is not being pulled with `ghcr.io/darpa-askem/terarium-theme:latest`?

Reply:
WIP: that package generated was a test, adding proper CI for this to happen now
additionally b/c its a `private` package will need everyone to login so adding documentation on how to do that

PR coming shortly